examples/all-clusters-app/all-clusters-common/src/binding-handler.cpp

```diff
@@ -63,12 +65,10 @@ static void RegisterSwitchCommands()
 {
     static const shell_command_t sSwitchCommand = { SwitchCommandHandler, "switch", "Switch commands. Usage: switch [on|off]" };
     Engine::Root().RegisterCommands(&sSwitchCommand, 1);
-    return;
 }
 #endif // defined(ENABLE_CHIP_SHELL)
 
-static void BoundDeviceChangedHandler(const EmberBindingTableEntry & binding, chip::OperationalDeviceProxy * peer_device,
-                                      void * context)
+static void BoundDeviceChangedFunc(const BindingTableEntry & binding, chip::OperationalDeviceProxy * peer_device, void * context)
```

Comment:
Why these name changes?  Non-substantive changes (like this renaming, and the namespace moves) should be in a separate PR from substantive changes (like changing the cluster architecture).  That would make the PR a _lot_ less time-consuming to review.

Stopping here.  Please split this PR up.

Reply:
Yes, I have changed this PR to only doing code move and using server cluster instead of AAI. And create https://github.com/project-chip/connectedhomeip/issues/40715 to follow up.